### PR TITLE
[Merged by Bors] - adding mastodon to community

### LINKED ISSUE #526: Add Bevy Mastodon Link to Community Page
https://bevyengine.org/community/ should include the https://mastodon.social/@bevy

### PR DESCRIPTION
**Pull request by vital14**
Fixes #526

### CODE REVIEW

**Comment by rparrett** [comment]
This is using the same icon as itch.io, could we find one for mastodon?

File: templates/community.html [hunk 1]
```diff
@@ -127,5 +127,21 @@
             </div>
         </div>
     </a>
+    <a class="card community-card" href=" https://mastodon.social/@bevy">
+        <div class="card-image">
+            <img src="/assets/itchio-textless.svg" class="centered-card-image community-icon" alt="itch.io logo" />
```
Comment (rparrett) [inline templates/community.html:132]: http://127.0.0.1:1111

File: templates/community.html [hunk 2]
```diff
@@ -127,5 +127,21 @@
             </div>
         </div>
     </a>
+    <a class="card community-card" href=" https://mastodon.social/@bevy">
```
Comment (rparrett) [inline templates/community.html:130]:
```suggestion
    <a class="card community-card" href="https://mastodon.social/@bevy">
```

File: templates/community.html (hunk 1 above)
Comment (vital14) [reply templates/community.html:132]: Yes, i where can i find it? i haven't changed because i don't know if it exists into the project

Comment (vital14) [reply templates/community.html:132]: Can i use an image from the internet? If  so, where do i add it into the project?

Comment (rparrett) [reply templates/community.html:132]: Maybe grab the "white mark-only" svg from here: https://joinmastodon.org/branding.

You can place is alongside the other icons in this repository.

File: templates/community.html [hunk 3]
```diff
@@ -127,5 +127,21 @@
...
+        </div>
+        <div class="card-text">
+            <div class="card-title">
+                mastodon
+            </div>
+            <div class="card-subtitle">
+                bevy
+            </div>
+            <div class="card-description">
+                Interact with us and community on mastodon.
```
Comment (alice-i-cecile) [inline templates/community.html:142]:
```suggestion
                News, boosted community creations and chatter on the short-form Fediverse network.
```

**Review by rparrett — APPROVED** [review]
Looks good to me!

**Review by alice-i-cecile — APPROVED** [review]
bors r+

**Pull request closed without merging**